src/pspm_cfg/pspm_cfg_tools.m

```diff
@@ -3,7 +3,7 @@
 cfg.name   = 'Tools';
 cfg.tag    = 'tools';
 cfg.values = {pspm_cfg_display, pspm_cfg_rename, pspm_cfg_split_sessions, ...
-    pspm_cfg_merge, pspm_cfg_artefact_rm, pspm_cfg_downsample, pspm_cfg_interpolate, ...
+    pspm_cfg_merge, pspm_cfg_artefact_rm, pspm_cfg_interpolate, ...
     pspm_cfg_extract_segments, pspm_cfg_segment_mean, pspm_cfg_get_markerinfo, ...
     pspm_cfg_data_editor, pspm_cfg_data_convert};
```

Comment:
`pspm_cfg_data_convert` does not exist. Shall we create it or remove `convert data` from Tools? I am confused about why it is there.